serverlessworkflow/modules/ROOT/pages/cloud/deploying-on-minikube.adoc

```diff
@@ -18,7 +18,7 @@ This document describes how to deploy your Serverless Workflow application using
 For more information about Minikube and related system requirements, see link:{minikube_url}/docs/start/[Getting started with Minikube] documentation.
 
 .Prerequisites
-* Your Serverless Workflow application is ready to use.
+include::../../pages/_common-content/getting-started-requirement.adoc[]
```

Comment:
not sure if it make sense here, wouldn't it duplicate things as it is on the quarkus cli guide as well.

Reply:
Yes. You're right. I'll revert it.